cfpb/college-costs
# Add verify info interactions with no JS

Added no-JS friendly dynamic interaction for settlement students to verify their information, or signify that it isn't their information.
## Additions
- Added `section` wrappers for `information-right` and `information-wrong` areas
- Added `target` selectors to use when showing and hiding verification instructions
## Changes
- Changed HTML verification `buttons` to `a` elements, so the `target` attribute could be used.
- Moved instructions for what to do if a student receives the wrong offer information lower in the HTML code
- Use semantic line breaks as advised in our front-end standards (https://github.com/cfpb/front-end/blob/master/markup.md#coding-style)
- Minor cleanup of lingering EOL spaces
- Updated functional tests to look for `a` elements for the verification buttons
## Testing
- Pull in the `no-js` branch and run `gulp`. Load it up, and try saying that it isn't your offer info, or that it is.
- Try it in standalone or in UnityBox. Try it on a phone or tablet.
- Try it in a browser with Javascript disabled, and see that it works there.
- Try it in IE8, to see that **it does not work there**. Luckily, the JS enhancement should, unless our IE8 user has also disabled JavaScript.

I tested this on Mac Chrome 47, Mac Safari 9.0, Mac Safari 9.0 with JS disabled, IE8, and Android Chrome Beta 48 on a Moto X Pure (Android 6.0).
## Review
- Any two of the following: @niqjohnson @ascott1 @mistergone
## Why not add this for the Big Question buttons in this PR?

Based on the way targeting works, I don't think the Big Question buttons can act the same way and maintain this setup. In addition, the financial calculations on the page currently require Javascript, so the Big Question and the core of tool itself isn't as useful without JavaScript.

This interaction doesn't require JavaScript, so it's good to code it from the start without. I've listed below in the Todos section some other ways to progressively enhance the experience.
## Related Todos
- Hook up the report to the school about faulty information
- Add JS to hide the button/link and some content
- Add CSS animation for a smoother interaction
- From a graphic design standpoint, look at removing/hiding the extra line that overlays the line that heads the footer. This irritates me visually, but I de-prioritized it for this PR.
## Screenshots

On load:

![screen shot 2016-01-07 at 11 20 47 am](https://cloud.githubusercontent.com/assets/1183435/12175521/c40df552-b530-11e5-973c-d9102b87b139.png)
## Checklist
